htdocs/expedition/class/expedition.class.php

```diff
@@ -373,13 +410,14 @@ function create_line($entrepot_id, $origin_line_id, $qty)
 	 * Create the detail (eat-by date) of the expedition line
 	 *
 	 * @param 	object		$line_ext		full line informations
+	 * @param	array		$array_options		extrafields array
 	 * @return	int							<0 if KO, >0 if OK
 	 */
-	function create_line_batch($line_ext)
+	function create_line_batch($line_ext,$array_options=0)
 	{
 		$error = 0;
 
-		if ($this->create_line(($line_ext->entrepot_id?$line_ext->entrepot_id:'null'),$line_ext->origin_line_id,$line_ext->qty) < 0)
+		if ($this->create_line(($line_ext->entrepot_id?$line_ext->entrepot_id:'null'),$line_ext->origin_line_id,$line_ext->qty,$array_options) < 0)
 		{
 			$error++;
 		}
```

Comment:
When I insert new extrafields on expedition line, method last_insert_id(...) returns expeditiondet rowid. I was not able to replicate your error. Can you give me more informations?